### DISCUSSION TITLE
Error getting zone fire forecast from zone ID

### DISCUSSION THREAD

**Discussion by danielg7** [weather-gov/api#740] [Q&A]
I'm trying to programmatically get the zone fire weather forecast for a given lat / lon.

I'm able to get the fire weather zone through there:

**Input**:
`https://api.weather.gov/zones?type=fire&point=40.5852%2C-105.0844&include_geometry=false`
**Output**:

> `{
>   "@context": {
>     "@version": "1.1"
>   },
>   "type": "FeatureCollection",
>   "features": [
>     {
>       "id": "https://api.weather.gov/zones/fire/COZ238",
>       "type": "Feature",
>       "geometry": null,
>       "properties": {
>         "@id": "https://api.weather.gov/zones/fire/COZ238",
>         "@type": "wx:Zone",
>         "id": "COZ238",
>         "type": "fire",
>         "name": "Larimer County Below 6000 Feet/Northwest Weld County",
>         "effectiveDate": "2024-03-05T18:00:00+00:00",
>         "expirationDate": "2200-01-01T00:00:00+00:00",
>         "state": "CO",
>         "forecastOffice": "https://api.weather.gov/offices/BOU",
>         "gridIdentifier": "BOU",
>         "awipsLocationIdentifier": "BOU",
>         "cwa": [
>           "BOU"
>         ],
>         "forecastOffices": [
>           "https://api.weather.gov/offices/BOU"
>         ],
>         "timeZone": [
>           "America/Denver"
>         ],
>         "observationStations": [],
>         "radarStation": null
>       }
>     }
>   ]
> }`

So my zone ID is: `COZ238`

If I go here: `https://api.weather.gov/zones/fire/COZ238` I get the geometry of the zone. That's great but not the forecast.

If I go here: `https://api.weather.gov/zones/fire/COZ238/forecast/ `
I get an error telling me the zone doesn't exist:

> `{
>     "correlationId": "17515f95",
>     "title": "Zone Does Not Exist",
>     "type": "https://api.weather.gov/problems/InvalidZone",
>     "status": 404,
>     "detail": "Forecast zone COZ238 does not exist",
>     "instance": "https://api.weather.gov/requests/17515f95"
> }`
> 

What am I doing wrong here?

**Comment by dwhitemv25** [comment]
Only forecast zones have forecasts issued for them through the `/forecast` endpoint. If you want the fire weather forecast, get product FWF from the WFO for the point. To do this in your case, extract the WFO code from the `forecastOffice` field, then fetch `https://api.weather.gov/products/types/FWF/locations/BOU` where BOU is the WFO code, then get the `@id` for the product issuance you want (most recent is first).

Is that the data you were looking for?

**Reply by dwhitemv25** [reply]
If you're looking for fire weather element forecasts (haines index, etc.), there are some available through the `/gridpoints` endpoint. The link is available through the `forecastGridData` property on the `/points` endpoint.
